mobile/src/main/java/com/truckmuncher/truckmuncher/authentication/AuthenticatorActivity.java

```diff
@@ -64,4 +70,13 @@ public void onLoginSuccess(String userName, String authToken) {
         setResult(RESULT_OK, intent);
         finish();
     }
+
+    @Override
+    public void onActivityResult(int requestCode, int resultCode, Intent data) {
+        super.onActivityResult(requestCode, resultCode, data);
+
+        if (fragment != null) {
+            fragment.onActivityResult(requestCode, resultCode, data);
```

Comment:
Is this call-through strictly required? Normally, if a fragment makes a `startActivity` call, the fragment will automatically receive the `onActivityResult` callback. However, idk how facebook does it's magic so this _may_ actually be required. just wanted to check.